tests/providers/amazon/aws/executors/ecs/test_ecs_executor.py

```diff
@@ -943,8 +944,10 @@ def test_provided_values_override_defaults(self, assign_subnets):
 
         assert task_kwargs["platformVersion"] == templated_version
 
-    def test_count_can_not_be_modified_by_the_user(self, assign_subnets):
+    @mock.patch.object(EcsHook, "conn")
+    def test_count_can_not_be_modified_by_the_user(self, mock_conn, assign_subnets):
         """The ``count`` parameter must always be 1; verify that the user can not override this value."""
+        mock_conn.describe_clusters.return_value = {"clusters": [{"status": "ACTIVE"}]}
```

Comment:
Was this test failing before this change?

Reply:
I guess you don't have to mock the return value itself, but you do need to mock the `conn` now that I added the describe_clusters call.  I can remove that line and it shouldn't make any real difference.

Reply:
Ah right, it's the describe clusters call that requires this. Got it. Carry on!

Reply:
I dropped the return value line.  I was thinking it was more clear with it in there, but it really only adds clutter to the test.  But yeah, the mock itself has to stay..